docs/source/guides/batching.rst

```diff
@@ -114,7 +114,7 @@ We will demonstrate this with a PyTorch example which accepts a dictionary of ``
                 torch.set_default_tensor_type("torch.cuda.FloatTensor")
             else:
                 self.device_id = "cpu"
-            self.model = model.load_model(device_id=self.device_id)
+            self.model = bentoml.pytorch.load_model(device_id=self.device_id)
```

Comment:
oh sorry, I think you also have to pass in the tag for the Runner, and then pass the tag via `runnable_init_params`.

Reply:
For a custom runnable for the usage of the service only, we can hardcode the tag here?

Reply:
Yeah, that would work.